<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPasswordDialog</class>
 <widget class="QDialog" name="ResetPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/photo_2024-07-06_16-47-18.jpg);
font: 700 15pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);

</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 15pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 15pt &quot;Segoe Print&quot;;
background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 15pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>phone number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phonenumber">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 15pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 15pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_newpassword">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 15pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>new password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newpassword">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 15pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
